style.css

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+body {
+  width: 372px;
+  background-color: #f7f7f9;
+}
```

Comment:
- [ ] According to projects requirements, you should show your knowledge of css `grid` property, therefore, the main container of the projects should have a `display:grid`. Try to apply your knowledge of `css grid` on this project 💪🏻